http-api/src/lib.rs

```diff
@@ -41,6 +43,13 @@ pub struct Options {
     pub theme: String,
 }
 
+#[derive(Serialize, Deserialize, Clone)]
+struct CommitsQueryString {
+    parent: Option<String>,
```

Comment:
Could we use `Option<One>` here? Then we wouldn't need to parse in the handler..

Reply:
Yeah we can use `Option<One>`, but we would still need to parse `meta.head` from a `git2::Oid` to a `One` in that case.
Or we could leave it as `Option<String>` and parse in the header to `One`